formidable/serializers/forms.py

```diff
@@ -28,7 +28,7 @@ class ConditionTestSerializer(serializers.Serializer):
 
 
 class ConditionSerializer(serializers.Serializer):
-    name = serializers.CharField(required=True)
+    name = serializers.CharField(required=False)
     fields_ids = serializers.ListField(
```

Comment:
is this argument required?

Reply:
It is not necessary, but "required=False" was used in another serializer, so I thought I could use it here to make it explicit